erpnext/hr/doctype/employee/employee.json

```diff
@@ -807,13 +815,117 @@
    "fieldtype": "Link",
    "label": "Shift Request Approver",
    "options": "User"
+  },
+  {
+   "fieldname": "passport_issue_date",
+   "fieldtype": "Date",
+   "label": "Passport Issue Date"
+  },
+  {
+   "fieldname": "passport_expiry_date",
+   "fieldtype": "Date",
+   "label": "Passport Expiry Date"
+  },
+  {
+   "collapsible": 1,
+   "fieldname": "passport_details_section",
+   "fieldtype": "Section Break",
+   "label": "Passport Details"
+  },
+  {
+   "fieldname": "column_break_84",
+   "fieldtype": "Column Break"
+  },
+  {
+   "collapsible": 1,
+   "fieldname": "residential_details_section",
+   "fieldtype": "Section Break",
+   "label": "Residential Details"
+  },
+  {
+   "fieldname": "country",
+   "fieldtype": "Link",
+   "label": "Country",
+   "options": "Country"
+  },
+  {
+   "fieldname": "residence_proof",
+   "fieldtype": "Attach",
+   "label": "Residence proof"
+  },
+  {
+   "description": "Example: Like QID in Qatar, Emirates ID in UAE, etc.",
+   "fieldname": "residential_id",
+   "fieldtype": "Data",
+   "label": "Residential ID"
+  },
+  {
+   "fieldname": "residential_expiry_date_if_applicable",
+   "fieldtype": "Date",
+   "label": "Residential Expiry Date (if Applicable)"
+  },
+  {
+   "fieldname": "passport",
+   "fieldtype": "Attach",
+   "label": "Passport"
+  },
+  {
+   "fieldname": "column_break_76",
+   "fieldtype": "Column Break"
+  },
+  {
+   "collapsible": 1,
+   "fieldname": "visa_details_section",
+   "fieldtype": "Section Break",
+   "label": "Visa Details"
+  },
+  {
+   "fieldname": "visa_type",
+   "fieldtype": "Select",
+   "label": "Visa Type",
+   "options": "Family\nWork Permit\nVisit Visa\nTemporary Work Permit"
```

Comment:
Could you change _Work Permit_ to **Employment Visa** and _Temporary Work Permit_ to **Temporary Employment Visa**, in UAE the Ministry of Labour also issues an ID called Work Permit which is used in WPS sif report